config/models/activity.js

```diff
@@ -13,7 +13,13 @@ var editionSchema = mongoose.Schema({
   link         : String,
   people       : [String], // Saved as NAME:ROLE
   start        : Date,
-  end          : Date
+  end          : Date,
+  enrolments   : Number,
+  participants : Number,
+  prizes       : Number,
+  budget       : Number,
```

Comment:
I don't agree with budget and I thought we said no it. It's an internal stuff and it doesn't represents the value of the project.